contexts/wallet.context.tsx

```diff
@@ -13,28 +14,47 @@ export function useWalletContext(): WalletInterface {
 }
 
 export function WalletContextProvider(props: PropsWithChildren<any>): JSX.Element {
-  const { wallets, selectedWallet } = useContext(BlueStorageContext);
-  const wallet = wallets.find((w: { getID: () => any }) => w.getID() === selectedWallet);
+  const { wallets, isElectrumDisabled, saveToDisk, sleep } = useContext(BlueStorageContext);
   const [address, setAddress] = useState<string>();
 
-  useEffect(() => {
-    if (wallet) {
-      setAddress(wallet.external_addresses_cache[0]);
+  function getWallet(): any {
+    return wallets[0];
+  }
```

Comment:
Why is this a method? 🤔

Reply:
had a different implementation first to get the correct wallet. with more knowledge it got reduced to this. But thinking about it, how is it not an issue? just accessing index 0 without any checks... 😂

Reply:
So maybe use `wallets?.[0]` you mean?

Reply:
yes totally